docusaurus.config.js

```diff
@@ -29,17 +29,34 @@ const config = {
   //     },
   //   ],
   // ],
+  // plugins: [
```

Comment:
If you evaluate the plugin is not needed then rather than comment more code it would be preferable to remove entirely